### PR TITLE
Fixes: #724 - Improved blog page

### PR DESCRIPTION
**Pull request by isuruAb**
Fixes issue #724 

Changes: 
- [x] Show full article
- [x] Make "author" linkable to FOSSASIA blog
- [x] Show linkable "tags" below article
- [x] Add Social Media share buttons

Demo Link: http://isuruab2.surge.sh/blog/

screen shot 2017-08-14 at 3 06 02 pm

@mariobehling  @rishiraj824  @uday96  @madhavrathi  please review this

### CODE REVIEW

**Comment by isuruAb** [comment]
@madhavrathi  please review again 👍

File: src/components/Blog/Blog.react.js [hunk 1]
```diff
@@ -35,39 +41,39 @@ class Blog extends Component {
...
-            'rss_url': 'http://blog.fossasia.org/tag/susi-ai/feed/',
-            'api_key': 'qsmzjtyycc49whsfvf5ikzottxrbditq3burojhd', // put your api key here
-            'count': 50
-        }
+            url: 'https://api.rss2json.com/v1/api.json',
+            method: 'GET',
+            dataType: 'json',
+            data: {
+                'rss_url': 'http://blog.fossasia.org/tag/susi-ai/feed/',
+                'api_key': 'qsmzjtyycc49whsfvf5ikzottxrbditq3burojhd', // put your api key here
```
Comment (rishiraj824) [inline src/components/Blog/Blog.react.js]: Is there a limit to this API ? OR this works?

**Comment by isuruAb** [comment]
@rishiraj824  please review it again. I removed scrolls.

File: src/components/Blog/Blog.react.js (hunk 1 above)
Comment (isuruAb) [inline src/components/Blog/Blog.react.js]: no there is no such a limit 👍

**Comment by uday96** [comment]
The scrolls are still there for the code part within the blog post, Please take a look: 

scer

**Comment by isuruAb** [comment]
@mariobehling  please review this http://isuruab2.surge.sh/blog/

**Comment by isuruAb** [comment]
@mariobehling please review this http://isuruab2.surge.sh/blog/

**Comment by mariobehling** [comment]
Thanks. Please open links in the same tab.

**Comment by mariobehling** [comment]
Links to categories are still not correct. They are now linking to a tag page instead of a category page.

**Comment by mariobehling** [comment]
You just changed the functionality when a user scrolls over a link. Please underline links in the text (not the bottom tags) and on scroll over change in the same way it works on the blog.

**Comment by mariobehling** [comment]
@isuruAb I don't understand some of the issues we are seeing here, e.g. the wrong links. Are you actually testing your own development? The quality of this simple commit is disappointing.

**Comment by mariobehling** [comment]
Pay attention to detail. Ensure categories stay categories and tags are tags.

**Comment by mariobehling** [comment]
Only link the actual word. Don't link the comma and following space in tags and categories.

**Comment by mariobehling** [comment]
Good job. Thanks.

**Pull request merged**